addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -348,5 +348,32 @@ def _get_summary_lines(self, templates):
             # We don't display deadlines when the user doesn't specify a plan_date
             if self.plan_date:
                 summary_line += f" ({format_date(self.env, template._get_date_deadline(self.plan_date))})"
+
+            # Add nested list for triggered and suggested next activities
```

Comment:
can you please elaborate on this comment?

Reply:
what I meant there is, we're already getting list of activity in plan summary, but that 'activity' also have some suggested/triggered activity which will be showing like a nested list.

**Plan Summary**

- Activity 1
  - Suggested Activity 1
  - Suggested Activity 2

Reply:
so can you please add that in the comment so that It is clear to all